### DISCUSSION TITLE
Output to HTTP stream or internet radio standard

### DISCUSSION THREAD

**Discussion by leinaht** [snapcast/snapcast#1181] [Q&A]
I'm trying to use an old Chumby as a Snapcast client via [HTTP streaming](http://forum.chumby.com/viewtopic.php?id=10241) or the built-in MP3/M3U/Ogg/PLS "radio" stream functions. Mopidy and MPD both have ways of outputting audio over HTTP, but that of course isn't synced with the Snapcast audio. Icecast seems to be natively supported but I have no experience with it and would rather stick with Snapcast.
I know this is a bit of a backwards use case but is there an easy way to set up a snapclient to then rebroadcast over HTTP? Maybe some kind of alsa loopback to feed it into something else?

**Comment by kingosticks** [comment]
Your "rebroadcast" is going to be out of sync so I don't see the benefit of doing this via snapcast. Just use mpd.

But if you must, then use snapclient's [file backend ](https://github.com/badaix/snapcast#client) to pipe the audio to something else that can send over HTTP (which will then be out of sync w.r.t to all other snapcast clients).
